frontend/src/modules/Survey/Styles/StepBegin.style.tsx

```diff
@@ -63,17 +61,25 @@ export const StyledFields = styled.div`
 export const StyledTitleWrapper = styled.div`
   display: flex;
   flex-direction: column;
+  width: 100;
 `
 
 export const StyledHeaderText = styled.text`
   ${h1};
   color: ${colors.black};
+  @media (max-width: 900px) {
+    font-size: 4rem;
+    margin-bottom: 10px;
+  }
 `
 
 export const StyledWelcomeText = styled.text`
   ${h2};
   font-weight: 300;
   color: ${colors.black};
+  @media (max-width: 900px) {
+    font-size: 2rem;
+  }
```

Comment:
Can some of this file be deleted now?

Reply:
🚮 yah I deleted what I think was safe so far to delete just now